crates/oxc_minifier/src/compressor/fold.rs

```diff
@@ -200,6 +201,9 @@ impl<'a> Compressor<'a> {
                     &binary_expr.left,
                     &binary_expr.right,
                 ),
+                BinaryOperator::Addition if self.options.evaluate => {
```

Comment:
Do you think it's good to add a comment explaining why "evaluate" only control `try_fold_addition` for now? I will be a little confused when I first read it without knowing the context.

Reply:
I agree, I've added a comment explaining what's going on